File: CPP/Mahasiswa.cpp
// Import library.
#include <iostream>
#include <string>

// Standard namespace.
using namespace std;

// Inisialisasi class.
class Mahasiswa
{
    // Atribut Private.
    private:
        string nama;
        string nim;
        string prodi;
        string fakultas;

    // Atribut Public.
    public:
        // Konstruktor.
        Mahasiswa()
        {
            // Set default nilai atribut.
            this->nama = "";
            this->nim = "";
            this->prodi = "";
            this->fakultas = "";
        }

        Mahasiswa(string nama, string nim, string prodi, string fakultas)
        {
            // Set nilai atribut.
            this->nama = nama;
            this->nim = nim;
            this->prodi = prodi;
            this->fakultas = fakultas;
        }

        // Getter dan Setter.
        // Setter.
        void set_nama(string nama)
        {
            this->nama = nama;
        }
        void set_nim(string nim)
        {
            this->nim = nim;
        }
        void set_prodi(string prodi)
        {
            this->prodi = prodi;
        }
        void set_fakultas(string fakultas)
        {
            this->fakultas = fakultas;
        }

        // Getter.
        string get_nama()
        {
            return this->nama;
        }
        string get_nim()
        {
            return this->nim;
        }
        string get_fakultas()
        {
            return this->fakultas;
        }
        string get_prodi()
        {
            return this->prodi;
        }

        // Method Remove Attribute.
        void hapusDataMahasiswa()
        {
            this->nama = "";
            this->nim = "";
            this->prodi = "";
            this->fakultas = "";
        }

        // Destruktor
        ~Mahasiswa()
        {
            
        }
};

File: Python/Mahasiswa.py
# Inisialisasi Class
class Mahasiswa:
    # Atribut mahasiswa
    __nama = ""
    __nim = ""
    __prodi = ""
    __fakultas = ""

    # Konstruktor
    def __init__(self, nama="", nim="", prodi="", fakultas=""):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # Setter
    def set_nama(self, nama=""):
        self.__nama = nama

    def set_nim(self, nim=""):
        self.__nim = nim

    def set_prodi(self, prodi=""):
        self.__prodi = prodi

    def set_fakultas(self, fakultas=""):
        self.__fakultas = fakultas

    # Getter
    def get_nama(self):
        return self.__nama

    def get_nim(self):
        return self.__nim

    def get_prodi(self):
        return self.__prodi
        
    def get_fakultas(self):
        return self.__fakultas